.github/workflows/pytest.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will install Python dependencies and run tests
+
+name: Python testing
+
+on:
+  push:
+    branches: [ master, rational ]
+  pull_request:
+    branches: [ master, rational ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Test with pytest
+      run: |
+        pytest . -s --cache-clear --ignore=tests --ignore=examples --doctest-modules
```

Comment:
This seems still to fail a couple of docstring tests

Reply:
If I am truly honest, I am not sure how to fix that. In the test one can see that the only discrepancy is the way sympy is writing the rational numbers. For some reason, in the GitHub action environment, the rational are printed as `MPQ(n,d)` instead of the simpler `n/d`.

I will look into it later.